### PR TITLE
en627 bottle filenames don't match .asc filenames

### PR DESCRIPTION
**Pull request by joannekoch**
Because en627 is the only cruise where bottle filenames don't match .asc filenames, the date, lat, lon,  and depth in nut_profile were set to NA.

Fix and made a special case when checking for en627.

### CODE REVIEW

File: neslter/parsing/nut/nut.py [hunk 1]
```diff
@@ -97,16 +97,18 @@ def merge_nut_bottles(sample_log_path, nut_path, bottle_summary, cruise):
...
     for file in sorted(glob(os.path.join(btl_dir, '*.asc'))):
+        if cruise == 'en627':
+            file = file.replace("_u", "")
         btl_file = file[:-3] + 'btl'
         if not os.path.exists(btl_file):
             _, cast = pathname2cruise_cast(btl_file)
             if cast is None:
                  continue
             cast = cast.lstrip('0')
             nut_profile.loc[nut_profile['cast'] == cast, 'date'] = 'NA'
```
Comment (joefutrelle) [inline neslter/parsing/nut/nut.py:108]: Should the date field be set to the empty string instead of NA in this case?

Comment (joannekoch) [reply neslter/parsing/nut/nut.py:108]: Yes, sorry the first pull request set the date string to the empty string, but I forgot to also include that change in the second pull request.

I could have just made one pull request, but I didn't know about the second pr when I made the first pr. Sorry for the confusion.

Comment (joefutrelle) [reply neslter/parsing/nut/nut.py:108]: Just pushed the equivalent change, looking at the conflict now.

Comment (joefutrelle) [reply neslter/parsing/nut/nut.py:108]: Botched that. Working on it ...

**Pull request merged**